packages/components/addon/components/hds/tabs/index.js

```diff
@@ -0,0 +1,82 @@
+import Component from '@glimmer/component';
+import { A } from '@ember/array';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+export default class HdsTabsIndexComponent extends Component {
+  @tracked tabNodes = A([]);
```

Comment:
@KristinLBradley I think you can use just plain arrays (I tried and it works).
See previous similar code here: https://github.com/hashicorp/design-system/blob/79047c1c088e1cbc40e85a89a92a108c75346ec1/packages/components/addon/components/hds/form/field/index.js#L13

Reply:
I was trying this and getting errors initially but I'll look at it again and check what you did in your branch, thanks.

Reply:
I was able to get this working now. (Replaced Ember array with plain JS one.)